mbf-site/package.json

```diff
@@ -55,6 +54,8 @@
     ]
   },
   "devDependencies": {
+    "@yume-chan/stream-extra": "^0.0.24",
```

Comment:
Oh, also, why is this still in the dependencies at all? Is there definitely a context where it's used as a dev dependency?

Reply:
I saved it to the wrong list...